/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/* 
 * File:   THomePage.h
 * Author: boutina
 *
 * Created on 16 d√©cembre 2015, 15:27
 */

#ifndef THOMEPAGE_H
#define THOMEPAGE_H

#include <QWidget>
#include <QMainWindow>
#include <QVector>
#include "../ui/ui_homepage.h"

class THomePage : public QMainWindow {
    Q_OBJECT
public:
    THomePage(QWidget *parent = 0);
    virtual ~THomePage();
private:
    void connectEvents();
    Ui::MainWindow ui;
};

#endif /* THOMEPAGE_H */

